components/locid/src/extensions/unicode/mod.rs

```diff
@@ -75,6 +75,7 @@ use litemap::LiteMap;
 /// ```
 #[derive(Clone, PartialEq, Eq, Debug, Default, Hash, PartialOrd, Ord)]
 #[allow(missing_docs)] // TODO(#1028) - Add missing docs.
+#[allow(clippy::exhaustive_structs)] // this type is stable
```

Comment:
Thought: As above: People shouldn't generally make one of these themselves, so `non_exhaustive` is safe, but I'm okay with it being exhaustive because it is backed by a spec that is unlikely to change in a breaking way.